modules/ai/assets/js/editor/module.js

```diff
@@ -135,6 +135,27 @@ export default class Module extends elementorModules.editor.utils.Module {
 			}
 		}
 
+		if ( 'excerpt' === aiOptions.type ) {
+			behaviors.ai = {
+				behaviorClass: AiBehavior,
+				type: aiOptions.type,
+				getControlValue: view.getControlValue.bind( view ),
+				setControlValue: ( value ) => {
+					if ( 'wysiwyg' === controlType ) {
```

Comment:
?

Reply:
this is how we update textual component with our generated text. see https://github.com/elementor/elementor/blob/9500508aa2579523674a60d9fa19f2b9711d7430/modules/ai/assets/js/editor/module.js#L78C36-L79C1